backend_new/prisma/schema.prisma

```diff
@@ -0,0 +1,1025 @@
+  reviewStatus                 ApplicationReviewStatusEnum   @default(valid) @map("review_status")
+  userId                       String?                       @map("user_id") @db.Uuid
+  listingId                    String?                       @map("listing_id") @db.Uuid
+  applicantId                  String?                       @unique() @map("applicant_id") @db.Uuid
+  mailingAddressId             String?                       @unique() @map("mailing_address_id") @db.Uuid
+  alternateAddressId           String?                       @unique() @map("alternate_address_id") @db.Uuid
+  alternateContactId           String?                       @unique() @map("alternate_contact_id") @db.Uuid
+  accessibilityId              String?                       @unique() @map("accessibility_id") @db.Uuid
+  demographicsId               String?                       @unique() @map("demographics_id") @db.Uuid
+  applicationFlaggedSet        ApplicationFlaggedSet[]
+  applicant                    Applicant?                    @relation(fields: [applicantId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  accessibility                Accessibility?                @relation(fields: [accessibilityId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  alternateContact             AlternateContact?             @relation(fields: [alternateContactId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  applicationsAlternateAddress Address?                      @relation("applications_alternate_address", fields: [alternateAddressId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  userAccounts                 UserAccounts?                 @relation(fields: [userId], references: [id], onUpdate: NoAction)
+  applicationsMailingAddress   Address?                      @relation("applications_mailing_address", fields: [mailingAddressId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  listings                     listings?                     @relation(fields: [listingId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  demographics                 Demographics?                 @relation(fields: [demographicsId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  preferredUnitTypes           UnitTypes[]
+  householdMember              HouseholdMember[]
+
+  @@unique([listingId, confirmationCode])
+  @@index([listingId])
+  @@map("applications")
+}
+
+// Note: [file_id, label] formerly max length 128
+model Assets {
+  id                            String              @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  createdAt                     DateTime            @default(now()) @map("created_at") @db.Timestamp(6)
+  updatedAt                     DateTime            @updatedAt @map("updated_at") @db.Timestamp(6)
+  fileId                        String              @map("file_id")
+  label                         String
+  listingEvents                 ListingEvents[]
```

Comment:
Am not seeing `listingEvents` or `listingImages` in `asset.entity`

Reply:
ah good question, this is one of the differences between prisma and typeorm 

in typeorm some relationships only need to live on one side (listingEvents had assests but assets didn't need to have listingEvents)

in prisma relations need to live on both sides 

so this is making explicit what used to be an implicit bijected relation